pkg/kclient/operators.go

```diff
@@ -25,7 +25,8 @@ const (
 
 // IsServiceBindingSupported checks if resource of type service binding request present on the cluster
 func (c *Client) IsServiceBindingSupported() (bool, error) {
-	return c.IsResourceSupported("binding.operators.coreos.com", "v1alpha1", "servicebindings")
+	return false, nil
+	//	return c.IsResourceSupported("binding.operators.coreos.com", "v1alpha1", "servicebindings")
```

Comment:
It might be good to mention `https://github.com/openshift/odo/issues/5084` in the comment to make anyone who is reading code in future understand why is this line commented out.